Ginger/GingerCoreNET/GeneralLib/Email.cs

```diff
@@ -355,6 +355,19 @@ public bool Send_SMTP()
                 
                 smtp.Send(myMail);
 
+                //clear and dispose attachments
+                if (myMail.Attachments != null)
+                {
+                    foreach (Attachment attachment in myMail.Attachments)
+                    {
+                        attachment.Dispose();
+                    }
+                    myMail.Attachments.Clear();
+                    myMail.Attachments.Dispose();
+                }
+                myMail.Dispose();
+                myMail = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'myMail'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5328154)